lib/sml-conv/conv.sig

```diff
@@ -1,15 +1,42 @@
+(* This signature should be implemented by any system planning on
+ * using this library. It just describes the terms in the system
+ * as well as the exception to be raised by a failing conversion.
+ *)
 signature CONV =
 sig
+  (* The type of things we're converting between *)
   type term
+
+  (* A conv is a function between two terms. It may be
+   * a partial function if the conversion is not
+   * applicable to all terms in which case it should
+   * raise the Conv exception
+   *)
   type conv = term -> term
   exception Conv
 end
 
+(* As a super set of the CONV signature this when
```

Comment:
I am not sure I understand the syntax of comment ;-)

Maybe something like "CONV_UTIL is an extension of the CONV signature which provides some convenient utility functions in case the terms are abstract binding trees."